### PR TITLE
chore: Update seed data with images and locations

### PR DESCRIPTION
**Pull request by louiechristie**
- [ ] This PR follows [contributing quidelines](CONTRIBUTING.md)
- [ ] This PR has tests (for backend in particular)

### What change does this PR introduce?

Add images and location seed data for the first three sharespots

### Notes

In a terminal run:

```console
docker-compose -f local.yml run django python manage.py dumpdata core.venue > core/fixtures/seed_data.json
```

Then afterwards images should show for the first three sharespot pages e.g.
http://localhost:8000/venue-detail/1

### Ticket

- [Trello Ticket](https://trello.com/c/xfwOLZig/120-add-images-and-location-data-for-the-first-three-sharespots)

### Screenshots

Screenshot 2021-01-31 at 17 40 50

### CODE REVIEW

**Comment by louiechristie** [comment]
@lili2311 Please can you review and merge? This also fixes the venue image so it's not hard-coded any more.

**Comment by tsvetelinak0** [comment]
I think there is another ticket (https://trello.com/c/Lq072Bb0/117-invite-everyone-to-the-admin-panel-in-production-staging) which gives us much easier access and way to populate our database. Would be great another opinion from others on that thought. I am a bit confused with all the commits pushed to that branch, would be nice to clean up a bit? :)

**Comment by louiechristie** [comment]
@tsvetelinak0 

> I think there is another ticket (https://trello.com/c/Lq072Bb0/117-invite-everyone-to-the-admin-panel-in-production-staging) which gives us much easier access and way to populate our database.

Seed data is useful for:
1. New members of the team setting up the project so they have initial data (or if you set up a new computer)
2. Copying data between environments: local, staging, production (but yes we could do it manually instead.)

> would be nice to clean up a bit?

I've tidied up the PR a bit. The npm scripts are just for convenience, because I got bored looking up and typing the long django script commands. Hopefully it looks clearer in the PR now. We can have a call to discuss.

File: README.md [hunk 1]
```diff
@@ -25,10 +25,15 @@ Run the tests with `pytest`
 1. **Terminal**: [iTerm2](https://www.iterm2.com/) (MacOSX), [Terminator](http://gnometerminator.blogspot.co.uk/p/introduction.html) (Linux) or use your preferred one.
 2. **Text Editor**: [Sublime Text](http://www.sublimetext.com/) or your preferred one.
 3. [Docker Desktop](https://www.docker.com/products/docker-desktop)
+4. [Node.js](https://nodejs.org/en/)
 
-## Project Setup
+## Getting started
 
-Open iTerm2 (or your preferred terminal.)
+Download and install [Docker Desktop](https://www.docker.com/products/docker-desktop) if you haven't already.
```
Comment (abu271) [inline README.md:32]: Should we mention [Docker Compose](https://docs.docker.com/compose/install/) installation here as well?

File: package.json [hunk 2]
```diff
@@ -5,9 +5,15 @@
...
         "tslint": "tslint --project tsconfig.json --format stylish --exclude **/src/**/*.js",
         "lint": "npm run tslint",
-        "dev": "webpack --mode development --watch"
+        "dev": "webpack --mode development --watch",
+        "dev:docker": "docker-compose -f local.yml up",
+        "makemigrations": "docker-compose -f local.yml run django python manage.py makemigrations",
+        "loaddata": "docker-compose -f local.yml run django python manage.py loaddata seed_data.json",
+        "dumpdata": "docker-compose -f local.yml run django python manage.py dumpdata core.venue > core/fixtures/seed_data.json",
+        "bootstrap": "npm i && npm run build:docker && npm run build && npm run makemigrations && npm run loaddata"
     },
```
Comment (abu271) [inline package.json:17]: Good idea :+1:  I always forget the Django commands

**Review by abu271 — APPROVED** [review]
This PR looks good to me.

File: README.md (hunk 1 above)
Comment (louiechristie) [reply README.md:32]: @abu271 good point. I've made the change, please can you approve it in PR https://github.com/PolyglotDevsLondon/sharespots/pull/120

**Pull request merged**